GitUI/GitUICommands.cs

```diff
@@ -736,7 +736,7 @@ private bool StartResetChangesDialog(string fileName)
                         }
                         else
                         {
-                            Directory.Delete(path, true);
+                            path.TryDeleteDirectory(out string _);
```

Comment:
We should display the error message here, too.

Reply:
This is only used from command line

Reply:
The existing code appears to be failing silently. I wasn't convinced a popup was necessary here.
I propose we add it if we have a real usecase

Reply:
Especially in FormCommit, operations on files have failed silently many times.
In FormCommit, one can see at least that it has not worked.
In FormBrowse, one must select the virtual commits in order to check the result.
I.e. bad UX.